corehq/apps/accounting/templates/accounting/version_summary_tab.html

```diff
@@ -16,7 +16,7 @@
                 </thead>
                 <tbody>
                     <tr>
-                        <td>{{ plan_version.product_rate.product.name }}</td>
+                        <td>{{ plan_version.product_rate }}</td>
```

Comment:
why isn't this `plane_version.product_rate.name`?

Reply:
```SoftwareProductRate``` has a custom implementation of ```__str__``` which gets used here.  It shows both the name and the monthly fee.